packages/react/src/themes/teams/withProcessedIcons.ts

```diff
@@ -12,21 +12,33 @@ const declareSvg = (svgIcon: SvgIconSpec, exportedAs?: string): ThemeProcessedIc
   exportedAs,
 })
 
-const processedIcons: ThemeIcons = Object.keys(svgIconsAndStyles as {
+const getIcon = iconAndMaybeStyles => {
+  return (iconAndMaybeStyles as any).styles
+    ? (iconAndMaybeStyles as SvgIconSpecWithStyles).icon
+    : (iconAndMaybeStyles as SvgIconSpec)
+}
+
+const processedIcons: { [key: string]: ThemeIconSpec } = Object.keys(svgIconsAndStyles as {
   [iconName: string]: TeamsProcessedSvgIconSpec
-}).reduce<ThemeIcons>((accIcons, iconName) => {
+}).reduce<{ [key: string]: ThemeIconSpec }>((accIcons, iconName) => {
```

Comment:
we can use `ThemeIcons` at both places where we have `{ [key: string]: ThemeIconSpec }` now - there is no need to use dictionary type. I would suggest to do one of the following:
- use `ThemeIcons` type here and for `processedIcons` above
- or remove `ThemeIcons` and use `ObjectOf<ThemeIconSpec>` at both places, if you think that this would be more readable

What do you think?

Reply:
Yep totally agree, at the first implementation the new names were required that is why I was changing these typings, but yes as they are optional now, I can use the same type. Good catch!